setup_arlon.sh

```diff
@@ -131,10 +127,8 @@ echo Arlon controller is up and running
 
 if ! which arlon &> /dev/null; then
     echo Downloading arlon CLI
-    wget -qc https://github.com/arlonproj/arlon/releases/download/v0.9.10/arlon_${arlon_os}_${arlon_arch}_0.9.10.tar.gz
-    tar -xf arlon_${arlon_os}_${arlon_arch}_0.9.10.tar.gz
-    mv arlon_${os}_${arch}_v0.9.10 ${HOME}/.local/bin/arlon
-    rm arlon_${arlon_os}_${arlon_arch}_0.9.10.tar.gz
+    make build
```

Comment:
Rest of the script doesn't assume Arlon code to be present. So the earlier script tries to setup Arlon without assuming that it is running from Arlon source code repo root. 
Whereas, this new version compiles Arlon CLI from source. What is the intended use of this script?

Reply:
Also, the script should check if kubectl is present / is available on the execution path. Because we attempt to apply manifests.

Reply:
> We should have an alias for the latest stable release and download that using wget/ curl from the script (unless we compile from source). If we are assuming that the source tree is present, the `version` file should be the correct source for the version to download.

This can be done. In that case there won’t be a need to run this make target from the root of the arlon repository

Reply:
> Rest of the script doesn't assume Arlon code to be present. So the earlier script tries to setup Arlon without assuming that it is running from Arlon source code repo root. Whereas, this new version compiles Arlon CLI from source. What is the intended use of this script?

This script  is for the end-user to setup arlon,argocd and the required services in their management cluster.

Reply:
> Also, the script should check if kubectl is present / is available on the execution path. Because we attempt to apply manifests.

We are already checking if kubectl is present and if not, then we are installing it